pkg/net/net.go

```diff
@@ -20,13 +20,17 @@ type NICCapability struct {
 	CanEnable bool   `json:"can_enable"`
 }
 
+type NICAddress struct {
+	PCI string `json:"pci"`
+	// TODO(fromani): add other hw addresses (USB) when we support them
+}
+
 type NIC struct {
 	Name         string           `json:"name"`
 	MacAddress   string           `json:"mac_address"`
 	IsVirtual    bool             `json:"is_virtual"`
 	Capabilities []*NICCapability `json:"capabilities"`
-	// TODO(jaypipes): Add PCI field for accessing PCI device information
-	// PCI *PCIDevice `json:"pci"`
+	HWAddress    NICAddress       `json:"hw_address"`
```

Comment:
I think I'd almost prefer to have a `*string` field called `PCIAddress` that is nullable to indicate the field isn't always filled in. If and when in the future we support non-PCI hardware addreses like USB, we can add another `*string` field called `USBAddress`.

Reply:
Works for me. I was never really fond of `hw_address`. It creates some confusion with the NIC physical address (MAC) which I prefer to avoid.